pkg/manager/manager.go

```diff
@@ -259,8 +260,37 @@ type Options struct {
 	// MetricsBindAddress is the TCP address that the controller should bind to
 	// for serving prometheus metrics.
 	// It can be set to "0" to disable the metrics serving.
+	// Per default metrics will be served via https and authenticated (via TokenReviews)
```

Comment:
With this change we now have two modes for metrics serving:
* secure (default): HTTPs + authentication/authorization
* insecure (old behavior): HTTP

We could also consider other variants, e.g.:
* authorized (default): HTTPs + authentication/authorization
* unauthorized: HTTPS

Reply:
Why change the default? Do we have any data for how common it is to want to authorize access to the metrics endpoint?

Reply:
The only data point I have is that in CAPI we basically had to disable metrics per default because of security concerns.

I'm absolutely fine with keeping the current default

Reply:
Updated